### ISSUE TITLE
Here's the 8chan popout code

### ISSUE THREAD

**Issue by fatchan** [vichan-devel/vichan#188]
http://8ch.net/js/youtube.js
"Released under the MIT license" -does that count if whoever is deving 8chan right now is not making updates public? haha

Some improvements and fixes made to it, feel free to add to vichan:
https://github.com/fatchan/vichan/commit/2d7cafe78e97f4c64d5cff6b9ab35ed1d2496f3b
-Fixed size so video no longer hags off bottom and is in the exact same place when the image is clicked
-It's already using font-awesome for the drag icon, so now the popout and unpop buttons use it
-Properly matching current theme

**Comment by sambuev** [comment]
How to use it with vichan?

**Comment by fatchan** [comment]
Simply replace vichan /js/youtube.js with my /js/youtube.js https://github.com/fatchan/vichan/blob/603228ca745188ec90ddd19d32584992fda58404/js/youtube.js

**Comment by fatchan** [comment]
Replace youtube.js with this one

**Comment by sambuev** [comment]
Okay, I will replace it, what is the benefits of using this script? 
Right now current vichan doesn't play videos on mobile devices, it uses flash-ebmded style and noone can play video from iOS devices. Did you fix this bug in your script?

**Comment by fatchan** [comment]
Not a bug but no, it hasn't been changed. Perhaps in mobile-style.js you could add a bit of jquery to get the src attribute of each video on the page, remove the video and add a link instead?

**Comment by sambuev** [comment]
Yes I have replaced it with your script, seems working okay, I can see youtube player and when I click it opens on iphone with no problem.
It's good, but VIMEO links still not playing on iOS devices. 

Please read this thread: 
[https://github.com/vichan-devel/vichan/issues/189](https://github.com/vichan-devel/vichan/issues/189)

**Comment by sambuev** [comment]
Do you know any other cool JS files which can be used with vichan?

**Comment by ctrlcctrlv** [comment]
> "Released under the MIT license" -does that count if whoever is deving 8chan right now is not making updates public? haha

It does yes. If you release a file with a certain license you aren't allowed to change your mind. If they forgot to remove that header, oh well, that sucks for them but is not your problem.

**Comment by ctrlcctrlv** [comment]
@j147 I don't know who you are or how you got pull access, but you might want to merge this.

**Comment by ctrlcctrlv** [comment]
@fatchan Have you tested this code with stock Vichan? I'm willing to merge it if so.

**Comment by Cipherwraith** [comment]
>If they forgot to remove that header, oh well, that sucks for them but is not your problem.

Its released under MIT; the license is not a mistake.

**Comment by panfu28** [comment]
http://8ch.net/js/youtube.js now redirects to another webpage but you can find the script mentioned on the issue on the wayback machine: https://web.archive.org/web/20190729224551/http://8ch.net/js/youtube.js
